io/ilvis2/Ilvis2Reader.cpp

```diff
@@ -50,14 +50,16 @@ std::string Ilvis2Reader::getName() const { return s_info.name; }
 Options Ilvis2Reader::getDefaultOptions()
 {
     Options options;
-    Option mapping("mapping", "LOW");
-    options.add(mapping);
+    options.add("mapping", "ALL", "The point to extract from the shot: LOW, HIGH, or ALL.  ALL creates 1 or 2 points per shot, depending on if LOW and HIGH are the same.");
```

Comment:
I *think* we're generally advertising fixed option values as lowercase, but it doesn't much matter.  Look around and see what you find.  I'm almost always forcing case when I check to avoid failures based only on case.

Also, in this case, probably best to use an enum internally, rather than a string.